### PR TITLE
Add anti-aliasing for android borders and backgrounds with rounded corners

### PR DESCRIPTION
**Pull request by PanayotCankov**

### CODE REVIEW

**Comment by enchev** [comment]
:+1:

File: ui/styling/background.android.ts [hunk 1]
```diff
@@ -90,72 +90,75 @@ export module ad {
...
+                    let imageHeight = params.repeatY ? bounds.height() : params.sizeY;
+                    params.posX = params.repeatX ? 0 : params.posX;
+                    params.posY = params.repeatY ? 0 : params.posY;
 
-                        canvas.drawRect(params.posX, params.posY, params.posX + w, params.posY + h, paint);
-                    }
+                    canvas.save();
+                    // Clipping here will not be antialiased but at least it won't shine through the rounded corners.
+                    // Path.Op can be used in API level 19+ to achieve the perfect geometry but at the moment we support 17-tish
+                    canvas.clipRect(params.posX, params.posY, params.posX + imageWidth, params.posY + imageHeight);
```
Comment (hshristov) [inline ui/styling/background.android.ts]: You can still use a check and apply `Path.Op' if API is 19+:
```
android.os.Build.VERSION.SDK_INT >= 19 // android.os.Build.VERSION.KITKAT but we don't have definition for it
```

**Comment by hshristov** [comment]
:+1:

**Comment by PanayotCankov** [comment]
run ci

**Comment by vakrilov** [comment]
:+1:

File: ui/styling/background.android.ts (hunk 1 above)
Comment (PanayotCankov) [inline ui/styling/background.android.ts]: I will

**Pull request merged**